fastlane/spec/actions_specs/carthage_spec.rb

```diff
@@ -423,6 +443,20 @@
         end.not_to raise_error
       end
 
+      it "works with cache_builds" do
+        expect do
+          Fastlane::FastFile.new.parse("lane :test do
+            carthage(
+              use_ssh: true,
+              use_submodules: true,
+              use_binaries: false,
+              cache_builds: true,
+              platform: 'iOS'
+            )
+          end").runner.execute(:test)
+        end.not_to raise_error
```

Comment:
i think the prev. test also covers this. so i am in favor of removing this particular test case.
as i would like to see this PR merged, @KrauseFx  whats your point?